beacon-chain/blockchain/service.go

```diff
@@ -234,16 +253,99 @@ func (c *ChainService) blockProcessing(processedBlock chan<- *types.Block) {
 	}
 }
 
+func (c *ChainService) blockProcessingNew(processedBlock chan<- *types.Block) {
```

Comment:
Can you make this the canonical method and rename the old one instead?

Reply:
Ok will do that